src/System.Devices.Gpio/System.Devices.Gpio/GPIOPin.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Devices.Gpio
+{
+    public enum GpioPinMode
+    {
+        Input,
+        Output,
+        InputPullDown,
+        InputPullUp
+    }
+
+    public enum GpioPinValue
+    {
+        Low = 0,
+        High = 1
+    }
+
+    public enum GpioEventKind
+    {
+        Low,
+        High,
+        SyncFallingEdge,
+        SyncRisingEdge,
+        AsyncFallingEdge,
+        AsyncRisingEdge
+    }
+
+    public enum GpioNumberingScheme
```

Comment:
`PinNumberingScheme` perhaps?

Reply:
or Mode or PinMode

Reply:
PinMode is already used for something else. Also I don't think the numbering scheme is a pin mode, it is just a way to identify a particular pin on the device. PinNumberingScheme makes more sense to me.